src/components/feed/feed.stories.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+import { Feed } from "./feed";
+
+/* 
+ @author [Jenny Doctor](https://github.com/jendoc)
+*/
+
+export default {
+  title: "Components/Feed",
+  component: Feed
+};
+
+
+export const Default = () => (
+  <Feed>
+    <Feed.Heading>Dec 07 (today)</Feed.Heading>
+    <Feed.Item>Notifications go here</Feed.Item>
```

Comment:
Could you include one that utilizes `Avatar` as well?

Reply:
Updated with notifications instead :) 
![Uploading Screenshot 2023-01-30 161958.png…]()